examples/lock-app/nrfconnect/main/gen/callback-stub.c

```diff
@@ -0,0 +1,2490 @@
+/**
```

Comment:
This file should not exist at all.  In fact, this directory doesn't exist and shouldn't exist.  This looks like a merge problem to me.